pasqal_cloud/client.py

```diff
@@ -36,7 +36,8 @@ class Client:
 
     def __init__(
         self,
-        project_id,
+        # Ticket (#622), this has to don't be optional anymore
```

Comment:
Fix phrasing

Reply:
I changed it in the last commit, was an amend though